src/main/java/org/broadinstitute/hellbender/engine/spark/GATKSparkTool.java

```diff
@@ -375,6 +383,36 @@ protected ReadFilter makeReadFilter(SAMFileHeader samFileHeader) {
         return Arrays.asList(new WellformedReadFilter());
     }
 
+    /**
+     * @see GATKTool#useVariantAnnotations()
+     */
+    public boolean useVariantAnnotations() {
+        return false;
+    }
+
+    /**
+     * @see GATKTool#getDefaultAnnotations()
+     */
+    public List<Annotation> getDefaultAnnotations() {
```

Comment:
Mirror the revised names from `GATKTool` here as well (`getDefaultVariantAnnotations()`, etc.)